libpod/container_api.go

```diff
@@ -538,7 +538,7 @@ func (c *Container) Unpause() error {
 	if c.state.State != define.ContainerStatePaused {
 		return errors.Wrapf(define.ErrCtrStateInvalid, "%q is not paused, can't unpause", c.ID())
 	}
-	defer c.newContainerEvent(events.Unpause)
+	defer c.NewContainerEvent(events.Unpause)
```

Comment:
I'm really hesitant to make these public; why do we need to add events outside of Libpod?

Reply:
Maybe we should move `pkg/autoupdate` into `libpod`? I want this to show up in the events.

Reply:
I'm still iffy because `generate systemd` isn't directly integrated with Libpod right now. We're relying on external annotations to determine what unit we're in, which is iffy.

If we're doing events, I would want to see both the old and new container IDs in the event

Reply:
Then I prefer reverting the events changes.

Reply:
Done.